lib/Menu.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'Menu'
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)